easybuild/easyconfigs/s/SeqAn/SeqAn-2.3.2-foss-2016b.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = "CMakeMake"
+
+name = 'SeqAn'
+version = '2.3.2'
+
+homepage = 'https://www.seqan.de/'
+description = """SeqAn is an open source C++ library of efficient algorithms and data structures
+ for the analysis of sequences with the focus on biological data"""
+
+# GCC > 5 required. https://github.com/seqan/seqan#prerequisites
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://github.com/seqan/seqan/archive/']
+sources = ['%(namelower)s-v%(version)s.tar.gz']
+
+# CMake > 3 required. CMake >= 3.4 recommended
+builddependencies = [('CMake', '3.4.3')]
```

Comment:
@pescobar We don't have an easyconfig for CMake 3.4.3 with `foss/2016b`, but we do for version `3.7.2`, so use that?